src/main/java/nl/knaw/dans/filemigration/core/FedoraToBagCsv.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2021 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.filemigration.core;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class FedoraToBagCsv {
+
+  private final String datasetId;
+  private final String doi;
+  private final String comment;
+  private final String type;
+  private final CSVRecord r;
+
+  private static final String DATASET_ID_COLUMN = "easyDatasetId";
+  private static final String DOI_COLUMN = "doi";
+  private static final String COMMENT_COLUMN = "comment";
+  private static final String TRANSFORMATION_TYPE_COLUMN = "transformationType";
+
+  public FedoraToBagCsv(CSVRecord r) {
+    datasetId = r.get(DATASET_ID_COLUMN);
+    doi = r.get(DOI_COLUMN);
+    comment = r.get(COMMENT_COLUMN);
+    type = r.get(TRANSFORMATION_TYPE_COLUMN);
```

Comment:
```suggestion
    transformation = r.get(TRANSFORMATION_TYPE_COLUMN);
```
`transformation` is a more meaningful name than `type`